WalletWasabi.Gui/Tabs/SettingsViewModel.cs

```diff
@@ -168,12 +179,17 @@ public override void OnOpen()
 				.ToProperty(this, x => x.IsPinSet, scheduler: RxApp.MainThreadScheduler)
 				.DisposeWith(Disposables);
 
-			Global.UiConfig.WhenAnyValue(x => x.LockScreenPinHash, x => x.Autocopy, x => x.IsCustomFee)
+			Global.UiConfig.WhenAnyValue(x => x.LockScreenPinHash, x => x.Autocopy, x => x.IsCustomFee, x => x.FeeDisplayFormat)
 				.Throttle(TimeSpan.FromSeconds(1))
 				.ObserveOn(RxApp.TaskpoolScheduler)
 				.Subscribe(async _ => await Global.UiConfig.ToFileAsync())
 				.DisposeWith(Disposables);
 
+			Global.UiConfig.WhenAnyValue(x => x.FeeDisplayFormat)
+				.ObserveOn(RxApp.MainThreadScheduler)
+				.Subscribe(x => FeeDisplayFormatString = ConvertFeeDisplayFormat(x))
+				.DisposeWith(Disposables);
```

Comment:
There's a circular call here. You change `FeeDisplayFormatString` for the change of `FeeDisplayFormat` and above you change `FeeDisplayFormat` for the change of `FeeDisplayFormatString`. This pattern often results in OSX freezes.

Reply:
What's the best way to do this?

Reply:
Not sure.

Reply:
This is fixed now, but thinking back shouldn't this not be a problem because we are setting the variable with `RaiseAndSetIfChange()` so it would only call if the value changed (so it only does 1 loop instead of infinite).